src/PrestaShopBundle/Bridge/Processor/ProccessFilter.php

```diff
@@ -0,0 +1,159 @@
+<?php
+
+namespace PrestaShopBundle\Bridge\Processor;
+
+use Context;
+use ObjectModel;
+use PrestaShop\PrestaShop\Adapter\LegacyContext;
+use PrestaShop\PrestaShop\Core\Hook\HookDispatcherInterface;
+use PrestaShopBundle\Bridge\Helper\HelperListConfiguration;
+use PrestaShopBundle\Bridge\Utils\CookieFilterUtils;
+use Symfony\Component\HttpFoundation\Request;
+use Tools;
+use Validate;
+
+class ProccessFilter
```

Comment:
```suggestion
class FiltersProcessor
```

Reply:
I think this naming is more fitting as it contains an action verb describing what the class does Because `ProcessFilter` sounds more like a DTO or container object in which you store the "filters used for process"

Reply:
Usually as a rule of thumb:
- methods do things => verb: `processFilter()`
- classes are things => name: `FilterProcessor`
- interfaces describe what things can do => adjective: `Serializable`, `Throwable`... or `FilterProcessorInterface` to keep the code more simple

Reply:
I agree your naming is better! I will update the class with this! Thank you!